continuous-pipe.yml.erb

```diff
@@ -220,3 +248,17 @@ tasks:
               limits:
                 cpu: 1
                 memory: 1G
+
+  migration:
+    run:
+      image:
+        from_service: web
+
+      commands:
+        - container setup
```

Comment:
I think we should fix this in the images or the specific seeds rather than run setup here when there's no filter to stop it running after first deployment